Loop.xcodeproj/project.pbxproj

```diff
@@ -4133,7 +4149,7 @@
 		4F7528911DFE1DC600C322D6 /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				APPLICATION_EXTENSION_API_ONLY = YES;
+				APPLICATION_EXTENSION_API_ONLY = NO;
```

Comment:
Was this an intentional change on the `Loop` project? If so, what was the reason?

Reply:
See above